config/__init__.py

```diff
@@ -200,14 +219,20 @@ def update_status_config(milestones_content):
     'type-webvr',
 ]
 
+# Get AB experiment variation values from the environement.
+# By default, it will serve v1 100% of the time.
+FORM_V1_VARIATION = os.environ.get('FORM_V1_VARIATION') or '0 100'
+FORM_V2_VARIATION = os.environ.get('FORM_V2_VARIATION') or '100 100'
+EXP_MAX_AGE = os.environ.get('EXP_MAX_AGE', None)
```

Comment:
max-age is defined as an integer so if we want to read this from the env var we should cast it to int (env vars are strings)